conf/services/pillar2-frontend.yml

```diff
@@ -0,0 +1,15 @@
+serviceName: Report Pillar 2 top-up taxes
+serviceDescription: |
+  Report Pillar 2 top-up taxes is a digital service whereby groups with a turnover of €750m or more can register and subscribe to the tax service and enable compliance to the global tax policy.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /report-pillar2-top-up-taxes
+contactFrontendServiceId: pillar2-frontend
```

Comment:
Can we check we have the correct value for contactFrontendServiceId ?

Reply:
If your service is not already integrating with contact-frontend, we advise choosing an identifier that is specific to your service and unlikely to be used by any other service, **avoiding any special characters** or whitespace.

Reply:
https://github.com/hmrc/accessibility-statement-frontend?tab=readme-ov-file#a-note-on-contactfrontendserviceid

Reply:
@javed-iqbal1 has confirmed that other services also for a dash '-' for their contactFrontendServiceId.
On that basis I'll approve this PR.